products/openzfs.md

```diff
@@ -19,21 +19,25 @@ eolColumn: Critical bug fixes
 releaseLabel: "OpenZFS __RELEASE_CYCLE__"
 releases:
 
-    - releaseCycle: "2.1"
-      release: 2021-07-02
-      eol: 2023-07-02
-      lts: true
-      latest: "2.1.4"
-
-    - releaseCycle: "2.0"
-      release: 2020-11-30
-      eol: 2021-12-23
-      latest: "2.0.7"
-
-    - releaseCycle: "0.8"
-      release: 2019-06-23
-      eol: 2020-12-14
-      latest: "0.8.6"
+-   releaseCycle: "2.1"
+    release: 2021-07-02
+    eol: 2023-07-02
+    lts: true
+    latest: "2.1.99"
+
+    latestReleaseDate: 2021-04-19
+-   releaseCycle: "2.0"
+    release: 2020-11-30
+    eol: 2021-12-23
+    latest: "2.0.7"
+
+    latestReleaseDate: 2021-12-23
+-   releaseCycle: "0.8"
+    release: 2019-05-21
```

Comment:
This is correct:
![image](https://user-images.githubusercontent.com/584253/169316275-4eedf7f8-d288-45e9-8a94-de5a4b7ffcf5.png)

https://github.com/openzfs/zfs/tags?after=zfs-2.0.0-rc5